properties-workitem/readme.md

```diff
@@ -0,0 +1,19 @@
+# Installation
+1. Git clone this repo
+2. Run `mvn clean install`
+3. Upload `target/properties-loader-<version>` to Business Central artifact repo
+4. Configure your PAM project settings:
+	* Dependencies > add `org.jbpm.process.workitem:properties-loader:<version>`  
+
+## PropertiesWorkItemHandler
+A work item that reads a java properties file from disk.
+### Install
+1. Add work item handler: `org.jbpm.process.workitem.properties.PropertiesWorkItemHandler` (Resolver: Reflection, NOT MVEL), add parameter `String directory` to point to directory where properties files are stored.  
+![picture](images/Deployments.png)
+2. Import WorkItemDefinition file `/src\main/resources/META-INF/PropertiesWorkItemDefinition.wid`
+3. Add dependency of type `org.jbpm.process.workitem:properties:0.0.3-SNAPSHOT`
+![picture](images/Dependencies.png)
```

Comment:
I think this can/should be done via internal workitems repository which will do that automatically now. So probably it should be changed to that form? Wdyt @mswiderski or @tsurdilo?

Reply:
+1. this is not needed as this workitem should be part of the workitem repository along all others.